### DISCUSSION TITLE
How to find the version of regolith-desktop

### DISCUSSION THREAD

**Discussion by gogsbread** [regolith-linux/regolith-desktop#947] [Q&A]
Probably, a noob question but I cannot deterministically find the major.minor version of the installed regolith desktop
- `apt info regolith-desktop` shows `4.4.2-1regolith-jammy` which clearly is not the version (probably needs fixing too)
-  None of the regolith packages have any version resembling 3.x
- `cat /etc/os-release` to no avail.

**Comment by w18g** [comment]
it's in the file:  /etc/regolith/version

or run:

regolith-diagnostic | head
